search/includes/classes/class-search.php

```diff
@@ -2022,4 +2092,8 @@ public function maybe_change_index_version() {
 			$this->versioning->set_current_version_number( $indexable, $normalized_version );
 		}
 	}
+
+	public function exclude_es_query_reserved_names( $taxanomies ) {
+		return array_merge( $taxanomies, self::ES_QUERY_RESERVED_NAMES );
```

Comment:
Could you please make it $tax**o**nomies? :-)

Reply:
Such a weird wor**l**d :)